message.go

```diff
@@ -186,6 +188,50 @@ type File struct {
 	Reader      io.Reader
 }
 
+// StickerFormatType is the file format of the Sticker.
+type StickerFormatType int
```

Comment:
I think it might be better to stick to just `StickerFormat` here, even though the API does call it `format_type`. I don't think losing the `Type` makes it any more ambiguous.